config/management/operational/src/governance.rs

```diff
@@ -1,218 +1,87 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{auto_validate::AutoValidate, rest_client::RestClient, TransactionContext};
-use aptos_global_constants::APTOS_ROOT_KEY;
-use aptos_management::{
-    config::{Config, ConfigPath},
-    error::Error,
-    secure_backend::ValidatorBackend,
-    transaction::build_raw_transaction,
-};
-use aptos_transaction_builder::aptos_stdlib as transaction_builder;
-use aptos_types::{
-    account_address::AccountAddress,
-    account_config::aptos_root_address,
-    chain_id::ChainId,
-    transaction::{authenticator::AuthenticationKey, ScriptFunction, TransactionPayload},
-};
+use crate::{auto_validate::AutoValidate, TransactionContext};
+use aptos_management::{config::ConfigPath, error::Error, secure_backend::ValidatorBackend};
+use aptos_types::{account_address::AccountAddress, chain_id::ChainId};
 use std::path::PathBuf;
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 pub struct CreateAccount {
     #[structopt(flatten)]
-    config: ConfigPath,
+    _config: ConfigPath,
```

Comment:
What is this?

Reply:
Legacy tooling that the blockchain team wants to keep while they migrate tests to the new CLI.